Makefile.common

```diff
@@ -343,6 +343,105 @@ $(eval $(call _OCAML_BYTECODE_PROGRAM,$(1)))
 $(eval $(call _OCAML_NATIVE_PROGRAM,$(1).opt))
 endef # OCAML_PROGRAM
 
+# Macros for OCaml libraries, similar to those for OCaml programs
+
+define _OCAML_LIBRARY_BASE
```

Comment:
This is identical to `_OCAML_PROGRAM_BASE` except that it defines an additional `_CMI_FILES`. That wouldn't hurt for the programs case as well, so could we instead have just the one `_OCAML_COMMON_BASE`?